compatibility-verifier/compCheck.sh

```diff
@@ -150,14 +150,15 @@ function setupCompatTester() {
 # Main
 #
 
-if [ $# -ne 1 ] ; then
+if [ $# -ne 2 ] ; then
   usage compCheck
 fi
 
 COMPAT_TESTER_PATH="pinot-integration-tests/target/pinot-integration-tests-pkg/bin/pinot-compat-test-runner.sh"
 
 # create subdirectories for given commits
 workingDir=$1
+testSuiteDir=$2
```

Comment:
compute the absolute path here, like I suggested in prev review

Reply:
done!